KachnaOnline.App/Controllers/EventsController.cs

```diff
@@ -144,30 +151,219 @@ public async Task<ActionResult<ManagerEventDto>> PlanEvent(BaseEventDto newEvent
+        /// </summary>
+        /// <param name="eventId">ID of the event to get the conflicting planned states for.</param>
+        /// <returns>A list of <see cref="EventDto"/>.</returns>
+        /// <response code="200">The list of conflicting states.</response>
+        /// <response code="404">No such event exists.</response>
+        [HttpGet("{eventId}/conflicting_states")]
+        [ProducesResponseType(typeof(IEnumerable<StateDto>), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> GetConflictingPlannedStatesForEvent(int eventId)
+        {
+            try
+            {
+                return this.Ok(await _eventsFacade.GetConflictingStatesForEvent(eventId));
+            }
+            catch (EventNotFoundException)
+            {
+                return this.NotFoundProblem("The specified event does not exist.");
+            }
+        }
+
+        /// <summary>
+        /// Get states linked to the event specified by <paramref name="eventId"/>.
+        /// </summary>
+        /// <param name="eventId">An ID of the event to link planned states to.</param>
+        /// <returns>An <see cref="EventDto"/> with list of <see cref="StateDto"/> as linked planned states.</returns>
+        /// <response code="200">The planned states were linked to the event.</response>
+        /// <response code="404">The event with the given ID <paramref name="eventId"/> does not exist.</response>
+        [AllowAnonymous]
+        [HttpGet("{eventId}/linked_states")]
+        [ProducesResponseType(typeof(IEnumerable<EventDto>), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> GetLinkedStatesForEvent(int eventId)
+        {
+            try
+            {
+                return this.Ok(await _eventsFacade.GetLinkedStatesForEvent(eventId));
+            }
+            catch (EventNotFoundException)
+            {
+                return this.NotFoundProblem("The specified event does not exist.");
+            }
+        }
+
+        /// <summary>
+        /// Link planned states to the event specified by <paramref name="eventId"/>.
+        /// </summary>
+        /// <param name="eventId">An ID of the event to link planned states to.</param>
+        /// <param name="plannedStatesToLinkDto">A list of planned state IDs to link to the event specified by <paramref name="eventId"/>.</param>
+        /// <returns>An <see cref="EventDto"/> with list of <see cref="StateDto"/> as linked planned states.</returns>
+        /// <response code="204">The planned states were linked to the event.</response>
+        /// <response code="404">The event with the given ID <paramref name="eventId"/> does not exist.</response>
+        /// <response code="403">The users is not an events manager.</response>
```

Comment:
Explicitly stating the 403 response code is redundant and inconsistent.

Reply:
Good to know. Removed.